app/helpers/application_form_builder.rb

```diff
@@ -105,7 +108,7 @@ def govuk_file_upload(attribute, label:, hint: nil)
     )
   end
 
-  def govuk_select(attribute, label:, items:, hint: nil)
+  def govuk_select(attribute, label:, label_classes: nil, items:, hint: nil)
```

Comment:
Default to empty string to avoid the need to call `to_s` later?

```suggestion
  def govuk_select(attribute, label:, label_classes: "", items:, hint: nil)
```